server/task/src/main/java/org/apache/james/task/Task.java

```diff
@@ -55,9 +55,10 @@ public Result onFailure(Operation... operation) {
             return this;
         }
 
-        private void run(Operation... operation) {
+        public Result run(Operation... operation) {
```

Comment:
that doesn't make sense to me. Could you explain this choice?

Reply:
What if you want to run something on success & on failure?

Reply:
Reverted (cf above)